src/types/types.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
I don't think you need `src/types/types.tsx`

I think `src/types.tsx` should be fine!